src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ CHANGELOG
    name as value, using it makes the command lazy
  * Added `cache:pool:prune` command to allow manual stale cache item pruning of supported PSR-6 and PSR-16 cache pool
    implementations
+ * Deprecated `Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader::loadTranslations` 
+   function, use `Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader::read` instead. 
```

Comment:
#23666 should be merged first so this is not needed (given the whole class is deprecated, you just introduce a new class with interface)